### PR TITLE
Complete UI booking field

### PR DESCRIPTION
**Pull request by nguyentrieutien1**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/73414)

## WHAT (optional)
- Create UI for booking field

## WHY (optional)

image
image

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'field_types/show'
```
Comment (manhhnee) [inline config/routes.rb:2]: ngoặc đôi anh ơi

**Review by manhhnee — COMMENTED** [review]
comment

**Comment by nguyentrieutien1** [comment]
ready

File: config/routes.rb (hunk 1 above)
Comment (nguyentrieutien1) [reply config/routes.rb]: wen

**Comment by nguyentrieutien1** [comment]
ready

File: app/controllers/field_types_controller.rb [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+  
+  def show; end
+  
+  def load_field_type
+    @field_type = FieldType.find_by(id: params[:id])
+    return if @field_type
+    
+    flash[:danger] = t("field_types.not_found")
+    redirect_to root_path
+
```
Comment (binhpt-1239) [inline app/controllers/field_types_controller.rb:14]: Bỏ line trống

**Comment by nguyentrieutien1** [comment]
ready

**Review by binhpt-1239 — APPROVED** [review]

**Pull request merged**